src/Upgrader/Business/PackageManager/Client/Composer/Json/Writer/ComposerJsonWriter.php

```diff
@@ -89,4 +89,12 @@ protected function adjustIndentation(string $encodedJson, int $indentation): str
     {
         return $this->printer->print($encodedJson, str_repeat(' ', $indentation));
     }
+
+    /**
+     * @return string
+     */
+    protected function getFileName(): string
```

Comment:
Why is this needed?

Reply:
For mock method in `ComposerJsonWriterTest`
Actual PR: https://github.com/spryker-sdk/upgrader/pull/2